spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/AutoId.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.firestore;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+//Copied from com.google.cloud.firestore.FirestoreImpl
```

Comment:
For new classes, add a top level Javadoc :) i.e. with the author tag and brief description etc.

Reply:
That's why I didn't want to introduce a new class :)

Reply:
Lol sorry for the inconvenience. At least it will be deleted shortly though! (Once they expose auto id).